### PR TITLE
Delta sync down for Android

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

**Comment by bhariharan** [comment]
Looks fine.

File: libs/SmartSync/src/com/salesforce/androidsdk/smartsync/phonegap/SmartSyncPlugin.java [hunk 1]
```diff
@@ -89,6 +90,7 @@ public void run() {
 		        		  case syncUp:             syncUp(args, callbackContext); break;
 		        		  case syncDown:		   syncDown(args, callbackContext); break;
 		        		  case getSyncStatus:	   getSyncStatus(args, callbackContext); break;
+                          case reSync:             reSync(args, callbackContext); break;
```
Comment (khawkins) [inline libs/SmartSync/src/com/salesforce/androidsdk/smartsync/phonegap/SmartSyncPlugin.java]: I think we need some code format templates for our IDEs. :)

**Comment by khawkins** [comment]
Looks fine. :+1:

File: libs/SmartSync/src/com/salesforce/androidsdk/smartsync/phonegap/SmartSyncPlugin.java (hunk 1 above)
Comment (wmathurin) [inline libs/SmartSync/src/com/salesforce/androidsdk/smartsync/phonegap/SmartSyncPlugin.java]: I've to check my tabs/spaces settings in Android Studio.

Comment (khawkins) [inline libs/SmartSync/src/com/salesforce/androidsdk/smartsync/phonegap/SmartSyncPlugin.java]: It's an issue we've all participated in at one time or another. :)  Perhaps we can move to something line the [IntelliJ Android code style](https://code.google.com/p/google-styleguide/source/browse/trunk/intellij-java-google-style.xml) when we cut over to Android Studio officially?  Cc: @bhariharan

**Pull request merged**